vip_routeid.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: vip-routeid-mappings
+  namespace: default
+data:
+  192.168.122.222: "64"
```

Comment:
This is similar to the problem that kube-keepalived-vip is running into with encoding too much information into a string mapping, so he's considering switching to a [CRD](https://github.com/aledbf/kube-keepalived-vip/pull/85). We don't have to do a CRD, but we should combine both mappings into a single yaml-based configmap. Yaml is hard to parse in bash, but much easier for end users.

```yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: kube-egress
data:
  mapping.yaml:
  - name: "foo"
    ip: "192.168.122.222"
    route: 64
    interface: "eth0"
    interval: 60
    services:
    - name: "myservice"
      namespace: "default"
    - name: "myservice2"
      namespace: "default"
  - name: "bar"
     ip: "192.168.122.223"
...
```